mitxgraders/helpers/calc/expressions.py

```diff
@@ -490,7 +388,7 @@ def parse_algebra(self):
             + Optional(suffix)("suffix")
```

Comment:
I highly approve of taking out the suffix matching at this stage, and just matching anything. I'm guessing that this will match something like `1.5e-3%`, which is now ambiguous? Also, will this match things like `1.5m%m%mmp`? Probably the price to pay in order to give a good error message...

Reply:
Re `1.5e-3%` Yes, that matches. And if `%` is defined as `0.01`, it evaluates to `1.5e-05` as it should.

Re `1.5m%m%mmp` Yes, that matches, but you do at least get a (somewhat) helpful error message: `'Invalid Input: m%m%mmp not permitted directly after a number`'

**But**: No useful error message is given for `5x_1`, since suffixes match `Word(alphas + '%')`. I briefly tried to make suffixes match all variables, but abandoned that due to time. Can come back to it later if we want, prefer to merge this without it.